blocks/header/header.css

```diff
@@ -42,10 +49,364 @@ header nav a:any-link {
+  content: "";
+  width: 12px;
+  display: inline-block;
+  height: 3px;
+  margin-bottom: 3px;
+  margin-left: 8px;
+  border-radius: 10px;
+  background-color: white;
+  border: 1px;
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul li span::after {
+  content: "";
+  width: 7px;
+  height: 2px;
+  top: 20px;
+  left: 15px;
+  position: absolute;
+  background-color: #169af3;
+  border-radius: 20px;
+  transform: rotate(-45deg);
+  border: 2px;
+}
+  
+header .nav-menu-overlay .nav-menu-column.right .highlight-link span::after
+{
+  content: "";
+  width: 1px;
+  margin-left: -6px;
+  padding: 2px;
+  height: 1px;
+  display: inline-block;
+  transform: rotate(225deg);
+  border-bottom: 3px solid white;
+  border-left: 3px solid white;
+  border-radius: 1px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .nav-menu-overlay-back::before {
+  content: "";
+  width: 3px;
+  position: absolute;
+  left: 13px;
+  padding: 3px;
+  height: 3px;
+  top: 14px;
+  transform: rotate(45deg);
+  border-bottom: 3px solid black;
+  border-left: 3px solid black;
+  border-radius: 1px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country{
+  width: 90%;
+  display: flex;
+  flex-direction: row;
+position: relative;
+  justify-content: start;
+  align-items: center;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country .country-list{
+  width: 280px;
+  top: 40px;
+  left: -160px;
+  z-index: 2;
+  background-color:  #001C2D;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country .country-list:has(a:hover){
+display: block !important;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country div{
+ position: relative;
+ 
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul li span {
+  display: inline-block;
+  min-width: 36px;
+  min-height: 36px;
+  border-radius: 50%;
+  position: relative;
+  margin-right: 15px;
+  margin-bottom: -8px;
+  background-color: #001c2d;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country span{
+  margin-top: 20px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right>ul {
+  font-size: 16px;
+  color: white;
+  line-height: 21.79px;
+  letter-spacing: 0.06em;
+  text-align: left;
+  display: flex;
+  padding: 0;
+  flex-direction: column;
+  gap: 15px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right>ul>li>ul {
+  font-size: 16px;
+  color: #bdccdb;
+  margin-top: 10px;
+  font-family: var(--body-font-family);
+  text-transform: none;
+  padding-top: 15px;
+  min-width: 100%;
+  border-top: 1px solid #004370;
+  line-height: 19.2px;
+  display: flex;
+  flex-direction: column;
+  gap: 13px;
+  font-weight: normal !important;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country div ul{
+ position: absolute;
+ padding-top: 30px !important;
+ padding-bottom: 15px !important;
+ top: 10px;
+ left: 10px;
+ border-radius: 20px;
+ display: none !important;
+ max-height: 200px;
+ overflow-y: scroll;
```

Comment:
Remove all `!important`. Try increasing the specificity of css if you want to make it work.

Reply:
Done